### DISCUSSION TITLE
Install Locally iPad Pro M4

### DISCUSSION THREAD

**Discussion by RAWHEEHOE** [open-webui/open-webui#3736] [Q&A]
i just purchased the new iPad Pro (M4 with 32GB RAM) and going to try and install OpenWebUI locally on to it (not using internet to a hosted service) - has anyone started this journey and have tips?

**Comment by justinh-rahb** [comment]
I don't know any (supported) method of running arbitrary code on iOS. This is going to be a non-starter.

**Reply by RAWHEEHOE** [reply]
i thought as much - i have done loads of research and love the method of https://github.com/guinmoon/LLMFarm so will contribute and build on this repository, should be fun ;)

**Reply by delfireinoso** [reply]
Even if LLFarm is fun to use even on a iPhone 12, and with 32 Gbytes of RAM you can run better models, this is a show project on possibilities of CoreML

The project is not as rich as OpenWebUi running at home accessed from my phone or iPad
